### PR TITLE
Update Cardinal to 2.2.5-4

### PR DESCRIPTION
**Pull request by jaxdesmarais**
### Summary of changes

- Update Cardinal frameworks in the SDK to 2.2.5-4
- Note: we finally have a .framework that works with x84_64 slices so can finally bump the .framework version as well! 🥳 

### Checklist

- [x] Added a changelog entry

- @scannillo @mattwylder @jaxdesmarais

### CODE REVIEW

**Review by sshropshire — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Braintree iOS SDK Release Notes
 
 ## unreleased
-* Adds support for Xcode 14 and iOS 16 
+* Adds support for Xcode 14 and iOS 16
+* Update Cardinal SDK to version 2.2.5-4
```
Comment (scannillo) [inline CHANGELOG.md:5]: take it or leave it comment:
```suggestion
* Update Cardinal SDK to version 2.2.5-4
  * Includes both Carthage `.framework` and `.xcframework` updates
```

**Review by scannillo — APPROVED** [review]

**Comment by scannillo** [comment]
Looks like we have failing 3DS unit and UI tests

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 * BraintreeSEPADirectDebit
   * Update nonce to pull in ibanLastFour and customerID as expected
```
Comment (scannillo) [inline CHANGELOG.md:7]: Random nitpick
```suggestion
  * Update `BTSEPADirectDebitNonce` to pull in `ibanLastFour` and `customerID` as expected
```

**Comment by jaxdesmarais** [comment]
> Looks like we have failing 3DS unit and UI tests

Yep we are looking into this 😢

**Comment by jaxdesmarais** [comment]
Closing due to Cardinal bug - we've reached out to them and will open a new PR once they have provided a way for us to resolve this issue.

**Pull request closed without merging**